content/en/docs/application-store/external-apps/deploy-clickhouse.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere"
+keywords: 'KubeSphere, Kubernetes, ClickHouse, ClickHouse Operator, ClickHouse Cluster'
+description: 'Learn how to deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere.'
+linkTitle: "Deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere"
+weight: 14340
+---
+
+[ClickHouse](https://clickhouse.tech/) is a column-oriented database management system (DBMS) for online analytical processing of queries (OLAP). ClickHouse on QingCloud is a ClickHouse cluster application deeply customized with the following features.
+
+- Automated cluster management. ClickHouse on QingCloud manages cluster status via the Paxos protocol. As you run ClickHouse on QingCloud and manage it like a single-node ClickHouse cluster, you will be free from complicated cluster management operations.
+- Data redistribution in clusters. Apart from automated management of the cluster, you can also perform cluster-level data operations, including `SYSTEM RESHARDING`, `SYSTEM RESHARDING DATABASE [database_name]`, and `SYSTEM RESHARDING TABLE [database_name].[table_name]`. Since you can redistribute your cluster data based on you needs, data query performance will be greatly optimized.
+- Excellent performance and less cost. ClickHouse on QingCloud adjusts certain key parameters and carries the excellent performance of ClickHouse clusters over into both single-node and multi-node cluster environments with various disk types supported. You can choose the disk type based on your needs and experience the excellent performance with no need to adjust any parameters.
```

Comment:
I am actually thinking if these paragraphs are appropriate as typically we do not put this kind of content in a guide. Even if they are included, there should not be too much of them.

Reply:
If that's the case, I think we can write down something like `ClickHouse on QingCloud is a ClickHouse cluster application deeply customized with features including automated cluster management, data redistribution in clusters, and excellent performance with less cost.`

Reply:
For the introduction part, refer to https://github.com/kubesphere/website/pull/1505.